ios-base/Helpers/Constants.swift

```diff
@@ -27,3 +28,26 @@ enum ErrorDomain: String {
   case generic = "GenericError"
   case parsing = "ParsingError"
 }
+
+struct UI {
+  enum Defaults {
+    static let margin: CGFloat = 32
+  }
+  
+  enum ViewController {
+    static let topMargin: CGFloat = 72
+  }
+  
+  enum Button {
+    static let cornerRadius: CGFloat = 22.0
+    static let height: CGFloat = 45.0
+    static let width: CGFloat = 200.0
+    static let spacing: CGFloat = 20.0
+  }
+  
+  enum TextField {
+  }
+  
+  enum Label {
+  }
```

Comment:
nitpick: Pls remove this if not used

Reply:
will be used in the followings PR when the rest of the views are to be implemented

Reply:
The overall recommendation is to keep the codebase as clean as possible and only add code that is used in the current version branch. This can be easily be added when it is needed since now it serves no purpose and have no members.